src/App.js

```diff
@@ -1,3 +1,18 @@
-export default function App() {
-  return <h1>Beads And Beyonds</h1>;
+import "./App.css";
+import { Navbar, Footer, Carousel, Aside, Category } from "./component";
+
+function App() {
+  return (
+    <div className="App">
+      <Navbar />
+      <Aside />
+      <main>
+        <Carousel />
```

Comment:
You can create a another component named Home.jsx and keep these mai. Thing inside that. It will help you to use the router things

Reply:
Okay I can do that